app/controllers/api/v1/events_controller.rb

```diff
@@ -50,4 +39,13 @@ def device_token
   def event
     event_params.merge(owner: user)
   end
+
+  def find_event
+    if event = Event.find(params[:id])
+      event
+    else
+      raise ActiveRecord::RecordNotFound,
+            "Couldn't find Event with 'id=#{params[:id]}"
```

Comment:
This is good. Try refactoring this with translation strings: http://guides.rubyonrails.org/i18n.html

You should be able to do something like:
```ruby
raise ActiveRecord::RecordNotFound,
      t('errors.events.not_found')
```